insert_local_user.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO "users" (username, hashed_password, email, full_name, disabled)
+VALUES ('johndoe', '$2b$12$niIeexAn3B2ASvQ6T3xJ3OJqsFkeGXj.hImNhiHxTwnxJoUezZy4m', 'johndoe@example.com', 'John Doe', false);
```

Comment:
I disagree with running raw SQL statements to insert data into the database.

I think we should create functionality to insert data into database via SQLAlchemy (The ORM).
Whilst this case is simple, it is not maintainable in the long term.